docs/topics/user_permissions.md

```diff
@@ -0,0 +1,51 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+
+In IoTP there are defined Services and SubServices. SubServices exists into Services.
+Tipicaly a Service represents a smartcity and all of the SubServices represents the verticals of that smartcity.
+
+Users are created into Services. The same user name could be used across different Services to represent different Users.
+i.e. adm1 could exists in "SmartCity" Service and adm1 could be another user for "SmartGondor" Service
+
+Roles are created into Services. By default all Services created into IoT platform are created with the following Roles:
+- ServiceCustomer: Role for a normal user of the Service, with standard capabilities.
+- SubServiceCustomer: Role for a normal user in a SubService, with standard capabilities.
+- SubServiceAdmin: Role for an administrator user in a SubService, with full capabilities.
+
+There is one and unique role common for all Services:
+- Admin: administrator with full capabilities of the Service
+
+Users receive roles assignments into Services (or Subservices). The permissions are determinated by the Role
+which a user has in a service (or subservice). Depending on the IoTP component, user permission implies the ability
+of do some actions or not.
+
+In deep details, each Role in a Service is defined by a Policy for each IoTP component.
+
+[IoTP Policies](https://github.com/telefonicaid/orchestrator/tree/master/src/orchestrator/core/policies)
+[Orion component actions](https://github.com/telefonicaid/fiware-pep-steelskin#-rules-to-determine-the-context-broker-action-from-the-request)
+
+
+## Can I modify permissions for a given user?
+The common way to modify permission for a given user is to assign or unassign roles.
+User can be assgined to admin, ServiceCustomer, SubServiceAdmin and SubServiceCustomer roles in a given Service or SubService
+This can be do it using portal and orchestrator.
```

Comment:
I'd suggest to capitalize terms that refers to IoT plafomr, i.e. Portal, Orchestrator...

By the way, not sure if the documentation is currently consistent with regards to references to Portal. What we dediced at the end @mrutid @mcarracedo ? To mention the Portal or not to mention?